src/test/PoolNestedActionsTest.sol

```diff
@@ -211,6 +216,19 @@ contract NestedActionExecutor is Test, PoolTestBase {
         _settle(key.currency1, user, int128(deltaThisAfter1), true);
     }
 
+    function _initialize() internal {
+        address locker = manager.getLocker();
+        assertTrue(locker != address(this), "Locker wrong");
+        PoolId id = key.toId();
+        (uint256 price,,) = manager.getSlot0(id);
+        assertEq(price, Constants.SQRT_RATIO_1_1);
```

Comment:
I think you can remove lines 222-224 given they are checking things about another pool. Better to have it check the pool in question before and after
- so before check that the price is 0
- and after check (as you have) that its `SQRT_RATIO_1_2`